lib/lotus/mailer.rb

```diff
@@ -66,7 +66,8 @@ def self.included(base)
         class_attribute :configuration
 
         self.configuration = conf.duplicate
-        self.templates(Hash.new)
+        self.templates = Hash.new
```

Comment:
I created another PR, please accept the other one before this one, the other has the templates changes without the attachment.